src/oemof/tabular/__init__.py

```diff
@@ -1,2 +1,13 @@
 __version__ = "0.0.4dev"
 __project__ = "oemof.tabular"
+
+
+import sys
+import warnings
+
+major, minor, micro = sys.version_info[:3]
+if minor < 9:
+    warnings.warn(
+        f"You are using python version {major}.{minor}.{micro}. oemof.tabular "
+        "will stop supporting python < 3.9 in the future."
```

Comment:
I would also use a `DepricationWarning` but somehow, this is no displayed in the console without setting `stacklevel=2`?!
```
    msg = f"You are using python version 3.8./n"
          "oemof.tabular will stop supporting python 3.8 in the future."
    warnings.warn(msg, DeprecationWarning, stacklevel=2)
```